### DISCUSSION TITLE
Rereun deployhook without reissuing cert

### DISCUSSION THREAD

**Discussion by rtadams89** [acmesh-official/acme.sh#5109] [Q&A]
Is there a way to trigger a deployhook to rerun using a certificate that was already issued/saved to disk from a previous run of acme.sh?

**Comment by bgibson710** [comment]
When you run the acme.sh --deploy command, it's just running your deploy hook, it is not reissuing the certificate.

**Comment by neilpang** [comment]
just use 
```
acme.sh --deploy  -d xxxxx   --deploy-hook xxxxx
```
